src/sentry/static/sentry/app/views/admin/adminProjects.tsx

```diff
@@ -1,32 +1,54 @@
-/* eslint-disable react/jsx-key */
 import React from 'react';
+import {Location} from 'history';
 import moment from 'moment';
 
 import ResultGrid from 'app/components/resultGrid';
 import {t} from 'app/locale';
+import {Organization} from 'app/types';
+import AsyncView from 'app/views/asyncView';
 
 export const prettyDate = function (x) {
   return moment(x).format('ll');
 };
 
-class AdminProjects extends React.Component {
-  getRow = row => [
-    <td>
+type Row = {
+  dateCreated: string;
+  name: string;
+  organization: Organization;
+  slug: string;
+  status: string;
+};
+
+type Props = {
+  location: Location;
+} & AsyncView['props'];
+
+export default class AdminProjects extends AsyncView<Props> {
```

Comment:
Missing state right?

Reply:
We aren't using the state in this component is it necessary?

Reply:
Can we make it functional in that case?